packages/bot-skeleton/src/scratch/blocks/Text/text_statement.js

```diff
@@ -32,8 +33,43 @@ Blockly.Blocks.text_statement = {
             description: '',
         };
     },
+    onchange(event) {
+        if (!this.workspace || this.isInFlyout || this.workspace.isDragging()) {
+            return;
+        }
+
+        const stack_blocks = Blockly.getMainWorkspace().getBlockById(event.blockId);
```

Comment:
```suggestion
        const stack_block = Blockly.getMainWorkspace().getBlockById(event.blockId);
```

Reply:
@nikolai-deriv this is an array of sub-blocks we use when we drag the whole stack